src/ux/step1/WalletProviders.tsx

```diff
@@ -22,6 +25,21 @@ const ProvidersWrapper = styled.div`
   padding: 8px;
 `
 
+const LanguageSelector = styled.select`
+  float: left;
+  border: 0;
+`
+
+const NoWalletFooter = styled.span`
+  float: right;
+  padding: 2px;
+  
+`
+const FooterWrapper = styled.div`
+  height: 35px;
+  padding: 0 8px 0 8px;
```

Comment:
Height should rarely be used. padding would be better:

```suggestion
  padding: 0 8px 25px 8px;
```